jsx/StaticDataTable.js

```diff
@@ -295,6 +295,68 @@ class StaticDataTable extends Component {
     });
   }
 
+  /**
+   * Get if the current sorting column has mixed types in its values.
+   *
+   * @return {bool} true if mixed types, else false.
+   */
+  hasMixedTypes() {
+    // TODO: data column type check should probably be done once at init,
+    // meaning when receiving data, to find which columns have mixed types.
+    let typesFound = null;
+
+    // not the default column
+    if (this.state.SortColumn === -1) {
+      return false;
+    }
+
+    // is the current sorted column with mixed type?
+    let isMixedType = false;
```

Comment:
Note that this variable is not necessary now that we are in a function, we can use `return x;` instead of `variable = x; break;`. I personally prefer returns over variables but some people may not agree so do as you want.

Reply:
Lets go with that.